src/main/groovy/life/qbic/datamodel/accounting/CostEstimate.groovy

```diff
@@ -61,4 +60,22 @@ class CostEstimate {
         this.totalPrice = totalPrice
         this.identifier = identifier
     }
+
+    /**
+     * Adds a new item to the items list of the offer
+     *
+     * @param item which should be added to current list of items
+     */
+    public void addItem(ProductItem item){
+        items.add(item)
+    }
+
+    /**
+     * Removes an item from the list of items of the offer
+     *
+     * @param item which should be removed from the current list of items
+     */
+    void removeItem(ProductItem item){
+         items.remove(item)
+    }
```

Comment:
Nice JavaDoc! I would argue that those methods belong to the super class `TomatoId` and are to be overwritten here only in case the behaviour differs to the super class. Quotation has the same methods.

Reply:
For the quotation and the cost estimate exists no superclass. Just for their IDs. And these functions are not part of the ID classes .. ;)

Reply:
Oh I see. Should there be a super class?